crates/topos-test-sdk/src/networking/mod.rs

```diff
@@ -0,0 +1,22 @@
+use std::net::SocketAddr;
+use std::net::{TcpListener, TcpStream};
+
+pub fn get_available_port() -> u16 {
+    get_available_addr().port()
+}
+
+pub fn get_available_addr() -> SocketAddr {
+    let host = "127.0.0.1";
```

Comment:
Is this changed to `127.0.0.1` on purpose?

Reply:
Yes I want to provide `addr` only for the loopback interface